test/test_neighborhood.py

```diff
@@ -0,0 +1,49 @@
+"""Testing the neighborhood module."""
+
+import pytest
+import toponetx as tnx
+
+import topoembedx as tex
+
+
+class TestNeighborhood:
+    """Test the neighborhood module of TopoEmbedX."""
+
+    def test_value_error(self):
+        """Testing if right assertion is raised for incorrect type."""
+        with pytest.raises(TypeError) as e:
+            tex.neighborhood.neighborhood_from_complex(1)
+
+        assert (
+            str(e.value)
+            == """Input Complex can only be a Simplicial, Cell or Combinatorial Complex."""
+        )
+
+    def test_matrix_dimensions_cellcomplex(self):
```

Comment:
According to the unit-test framework, the name of the test function has to be test_[name of the function being tested].

here: test_neighborhood_from_complex.